build.gradle

```diff
@@ -996,6 +1020,10 @@ def getProjectVersion() {
     return ver
 }
 
+def getDefaultJavaVersion() {
+    return JavaVersion.VERSION_11.getMajorVersion()
```

Comment:
why this complexity ? istn't "11" enough ? you are still hardcoding "VERSION_11" here

Reply:
Yeah, I debated that. Definitely unnecessary.